packages/jest-runner/src/index.ts

```diff
@@ -180,7 +181,18 @@ class TestRunner {
       ),
     );
 
-    const cleanup = () => worker.end();
+    const cleanup = async () => {
+      const {forceExited} = await worker.end();
+      if (forceExited) {
+        console.log(
```

Comment:
stderr instead? not sure

Reply:
Dunno, last time I added a warning was `jest-circus/jest-jasmine2` which also used `console.log(chalk.yellow())`. You know the code base better than I do, what is more commonly used for warnings? 😄